utils/timestamp.py

```diff
@@ -67,6 +70,7 @@ def plot_pts(diffs, avg, title):
     parser.add_argument('filename', help='PTS file generated from rpicam-vid (with a .txt or .pts extension)'
                                          ' or an avi/mkv/mp4 container file', type=str)
     parser.add_argument('--plot', help='Plot timestamp graph', action='store_true')
+    parser.add_argument('--narrow', help='Frameskips outside view', action='store_true')
```

Comment:
Could this description be changed to something like "Narrow the y-axis by hiding outliers" or similar please?

Reply:
@naushir I did the changes you requested, and followed instructions regarding <kbd>git commit --amend</kbd>:
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/changing-a-commit-message

I got into trouble with merge conflicts, but finally got problems resolved.
The new hash shows both requested changes, to commit message and line 73 of timestamp.py:
https://github.com/Hermann-SW/rpicam-apps/commit/0fe3a4209c96fa62e23d22a45e5c0fbb0d55af14